cmd/tailscale/cli/configure.go

```diff
@@ -17,8 +17,10 @@ var configureCmd = &ffcli.Command{
 	Name:      "configure",
 	ShortHelp: "Configure the host to enable more Tailscale features",
```

Comment:
```suggestion
	ShortHelp: "[ALPHA] Configure the host to enable more Tailscale features",
```
We don't have guidance written down, but I believe this is consistent with cmd/tailscale/cli/update.go